compiler/src/dotty/tools/dotc/core/classfile/ClassfileParser.scala

```diff
@@ -750,6 +750,7 @@ class ClassfileParser(
           in.skip(attrLen)
           // flag test will trigger completion and cycles, thus have to be lazy
           if (sym.owner.flagsUNSAFE.isAllOf(Flags.JavaInterface)) {
+            sym.setFlag(Flags.DefaultMethod)
```

Comment:
It's not clear to me that we need this flag at all, it's checked in the semanticdb logic but I don't think we emit semanticdb for java sources so that's likely dead code. It's also checked in JavaParsers but it's only needed while parsing a method, so not something worth using up one of the 64 slots in Flags.scala for.

Reply:
I needed it to forward port the improved "stubs" messaging in refchecks (where it generates the methods missing implementations). I don't remember offhand what it missed, but I can revive the branch. I cherry-picked this to see if it would ring a bell.

Reply:
I see I had debugging in semanticdb but I don't recall what I was doing over there.